### PR TITLE
Add alpha 5 changelog.

### PR DESCRIPTION
**Pull request by vpetrovykh**

### CODE REVIEW

**Comment by vpetrovykh** [comment]
I'm using `Luhman 16` as the release name because it appears to be next in the list of closest stars after Barnard's (https://simple.wikipedia.org/wiki/List_of_nearest_stars).

**Comment by vpetrovykh** [comment]
Also, I assume that the versions for `edgedb-pyhton` and `edgedb-js` will be bumped to these since the current ones are 0.9.0.

File: docs/changelog/1_0_a5.rst [hunk 1]
```diff
@@ -0,0 +1,60 @@
+===========
+1.0 Alpha 5
+===========
+
+This changelog summarizes new features and breaking changes in
+EdgeDB 1.0 alpha 5 "Luhman 16".
```
Comment (ambv) [inline docs/changelog/1_0_a5.rst:6]: I'd remove the "16" as it looks confusing with all the previous numbers.

Comment (1st1) [reply docs/changelog/1_0_a5.rst:6]: +1

File: docs/changelog/1_0_a5.rst [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  (:eql:gh:`#251`).
+* Allow constraints on tuple types (:eql:gh:`#1576`).
+* Allow constraints directly on object types in SDL (:eql:gh:`#1164`)
+* Disallow using :eql:constraint:`exclusive` on scalar types
+  (:eql:gh:`#1575`).
+* Proper implementation of ``SET/DROP OWNED``.
+* Fix issues with some ``FOR`` statements (:eql:gh:`#1594`).
+* Use fully-qualified names to disambiguate the expressions produced
+  by :eql:stmt:`DESCRIBE` (:eql:gh:`#1254`).
+* Initial implementation of ``INSERT ... ON CONFLICT ... ELSE``
```
Comment (1st1) [inline docs/changelog/1_0_a5.rst:22]: I'd expand with a few examples here and there. That would make the changelog more interesting to read. This one for example would benefit from that.

File: docs/changelog/1_0_a5.rst [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+
+
+GraphQL
+=======
+
+* Allow several mutation operations in a single mutation query
+  (:eql:gh:`#1569`).
+* Reflect nested aliased types (:eql:gh:`#722`).
+* Enable sorting on non-trivial path (:eql:gh:`#1642`).
+* Add an ``exists`` filter operation (:eql:gh:`#1655`).
```
Comment (1st1) [inline docs/changelog/1_0_a5.rst:35]: This also deserves an example

File: docs/changelog/1_0_a5.rst [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+
+CLI
```
Comment (1st1) [inline docs/changelog/1_0_a5.rst:38]: Add "JavaScript binding" section and mention https://github.com/edgedb/edgedb-js/commit/fe5c1a605683ede999c367ff725bececfcebeccc

Comment (1st1) [reply docs/changelog/1_0_a5.rst:38]: With an example also

File: docs/changelog/1_0_a5.rst [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+  (`RFC 1000 <migrations_>`_).
+
+
+GraphQL
+=======
+
+* Allow several mutation operations in a single mutation query
+  (:eql:gh:`#1569`).
+* Reflect nested aliased types (:eql:gh:`#722`).
+* Enable sorting on non-trivial path (:eql:gh:`#1642`).
```
Comment (1st1) [inline docs/changelog/1_0_a5.rst:34]: Add an example

File: docs/changelog/1_0_a5.rst [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+* Allow constraints on tuple types (:eql:gh:`#1576`).
+* Allow constraints directly on object types in SDL (:eql:gh:`#1164`)
+* Disallow using :eql:constraint:`exclusive` on scalar types
+  (:eql:gh:`#1575`).
+* Proper implementation of ``SET/DROP OWNED``.
+* Fix issues with some ``FOR`` statements (:eql:gh:`#1594`).
+* Use fully-qualified names to disambiguate the expressions produced
+  by :eql:stmt:`DESCRIBE` (:eql:gh:`#1254`).
+* Initial implementation of ``INSERT ... ON CONFLICT ... ELSE``
+  (:eql:gh:`#1369`)
```
Comment (ambv) [inline docs/changelog/1_0_a5.rst:23]: That's actually 1639.

File: docs/changelog/1_0_a5.rst [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+* Add ``edgedb server status --all`` command to list all instances.
+
+
+Misc
+====
+
+* Add support of connecting to instance by a name (`#112
+  <https://github.com/edgedb/edgedb-python/pull/113>`_).
+* Update the `edgedb-js <https://github.com/edgedb/edgedb-js>`_ driver
+  to v0.10.0.
```
Comment (ambv) [inline docs/changelog/1_0_a5.rst:53]: Looks like we are 0.9.0 on edgedb-js:

https://www.npmjs.com/package/edgedb

Comment (1st1) [reply docs/changelog/1_0_a5.rst:53]: Yes, 0.9.0 is the latest for edgedb-js, and 0.10.0 is the latest for edgedb-python.

**Comment by 1st1** [comment]
@vpetrovykh We need this in before tomorrow noon PT. If you won't have time to address the comments we'll merge as is.

File: docs/changelog/1_0_a5.rst (hunk 1 above)
Comment (vpetrovykh) [reply docs/changelog/1_0_a5.rst:6]: Done

File: docs/changelog/1_0_a5.rst (hunk 4 above)
Comment (vpetrovykh) [reply docs/changelog/1_0_a5.rst:38]: Our bindings-related info appeared in "Misc" category last time, so I can rename that to "Bindings" and add the transactions in js there.

File: docs/changelog/1_0_a5.rst (hunk 7 above)
Comment (vpetrovykh) [reply docs/changelog/1_0_a5.rst:53]: Fixed.

File: docs/changelog/1_0_a5.rst (hunk 6 above)
Comment (vpetrovykh) [reply docs/changelog/1_0_a5.rst:23]: Fixed

File: docs/changelog/1_0_a5.rst (hunk 2 above)
Comment (vpetrovykh) [reply docs/changelog/1_0_a5.rst:22]: I'll link it to the relevant docs section for `INSERT` instead. See #1706

File: docs/changelog/1_0_a5.rst (hunk 3 above)
Comment (vpetrovykh) [reply docs/changelog/1_0_a5.rst:35]: Added.

File: docs/changelog/1_0_a5.rst (hunk 5 above)
Comment (vpetrovykh) [reply docs/changelog/1_0_a5.rst:34]: Added.

**Pull request merged**